docs/client-options.md

```diff
@@ -118,6 +118,7 @@ third party clients.
 - [Certify](https://github.com/webprofusion/Certify) GUI (.NET, WinForms)
 - [oocx/acme.net](https://github.com/oocx/acme.net) (.NET)
 - [kelunik/acme-client](https://github.com/kelunik/acme-client) (PHP)
+- ZeroSSL: [Portable Perl version](https://zerossl.com/installation.html#portablePerl), [Portable Win32/Win64 binaries](https://github.com/do-know/Crypt-LE/releases).
```

Comment:
The Perl version is already listed. Could you update this to be just:
` - [ZeroSSL Win32/Win64](https://github.com/do-know/Crypt-LE/releases)`